base/dotenv.mustache

```diff
@@ -2,4 +2,4 @@ JWT_SECRET=foo
 RETHINK_HOST=localhost
 RETHINK_PORT=28015
 RETHINK_NAME={{name}}
-PORT=3001
+PORT=4678
```

Comment:
Looks like this is an accidental inclusion into this PR. Please remove and squash.